TCPClient.pro

```diff
@@ -4,7 +4,10 @@
 #
 #-------------------------------------------------
 
+PATH_TO_SOURCE = /home/liux0/beamline/programming/AcquamanDataServer/source
```

Comment:
Will these also need to be generalized for testing on the server? 
(My guess is that although the TCPClient won't be 100% required in order to get the server running down here, and interfacing with Acquaman, it will never-the-less prove a useful diagnostic tool)